FeedStoreChallenge/Core Data/CoreDataFeed.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CoreDataFeed.swift
+//  FeedStoreChallenge
+//
+//  Created by OLEKSANDR ISAIEV on 13.05.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataFeed)
+final class CoreDataFeed: NSManagedObject {
+	@NSManaged public var timestamp: Date
+	@NSManaged public var coreDataFeedImages: NSOrderedSet
+}
+
+extension CoreDataFeed {
+	var localFeed: [LocalFeedImage] {
+		return coreDataFeedImages
+			.compactMap { $0 as? CoreDataFeedImage }
+			.map { $0.local }
+	}
+	
+	static func managedFeedSet(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```

Comment:
`CoreDataFeed` uses `images` identifier to address an ordered set of `CoreDataFeedImage`. Thus it is better to call it `managedImagesSet`. Moreover, since the context is clear, it can be called simpler, just `images`.

Please rename `managedFeedSet` to `images`.

```suggestion
	static func images(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```